### PR TITLE
New Game: Catch_The_Falling_Objects

### LINKED ISSUE #4311: [New game]: Catch the Falling Object
### 🎮 Game Request

Game logic and basic description : 
Objects will be falling from the top and we have to collect that at the bottom
Screenshot 2024-06-07 122438

### Point down the features

- Score will be displayed.
- 5 lives will be provided to the player.
- If player will not catch the objects, then player will loose 1 life 

### Select program in which you are contributing

GSSoC24

### Code of Conduct

- [X] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) of this project.

### PR DESCRIPTION
**Pull request by katarianikita2003**
New Game: Catch_The_Falling_Objects

Game logic and basic description :
Objects will be falling from the top and we have to collect that at the bottom

Fixes  #4311 

Point down the features
-Score will be displayed.
-5 lives will be provided to the player.
-If player will not catch the objects, then player will loose 1 life
<hr>
 
## Mark the task you have completed ✅

- [x] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) & [CODE OF CONDUCT](https://github.com/kunjgit/GameZone/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I have followed proper naming convention showed in [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md)
- [x] I have added screenshot for website preview in assets/images 
- [x] I have added entries for my game in main README.md
- [x] I have added README.md in my folder 
- [x] I have added working video of the game in README.md (optional)
- [x] I have specified the respective issue number for which I have requested the new game.

<hr>

Catch The Falling Objects 1

Catch The Falling Objects 2

https://github.com/kunjgit/GameZone/assets/158465309/6e752df8-c658-45ff-bbe4-56024da6ae2f

--- 
<br>

### CODE REVIEW

**Comment by katarianikita2003** [comment]
Heyy @kunjgit , please review it.

**Comment by katarianikita2003** [comment]
@kunjgit , please review it, either merge it or close it so that i can work on another issue

File: README.md [hunk 1]
```diff
@@ -234,13 +234,14 @@ This repository also provides one such platforms where contributers come over an
 | [Dragon_Tower](https://github.com/kunjgit/GameZone/tree/main/Games/Dragon_Tower)               |
 | [Hover_Board_Effect](https://github.com/kunjgit/GameZone/tree/main/Games/Hover_Board_Effect)   |
 [Mancala_Game](https://github.com/kunjgit/GameZone/tree/main/Games/Mancala_Game)                                                  |
-| [Dice_Roller](https://github.com/kunjgit/GameZone/tree/main/Games/Dice_Roller)                  | [Bear Hunter Ninja](https://github.com/Niyatizzz/GameZone/tree/main/Games/Bear_Hunter_Ninja) | 
-| [Rock_paper_scissor](https://github.com/kunjgit/GameZone/tree/main/Games/Rock_paper_scissor)   | 
-| [City_Builder_Game](https://github.com/kunjgit/GameZone/tree/main/Games/City_Builder_Game)     | 
-| [Chrome_Dino_Game](https://github.com/kunjgit/GameZone/tree/main/Games/Chrome_Dino_Game)       | 
-| [Pokemon_Stats_Card](https://github.com/kunjgit/GameZone/tree/main/Games/Pokemon_Stats_Card)   |
+| [Knife_hit](https://github.com/kunjgit/GameZone/tree/main/Games/Knife_hit)       |
```
Comment (ishita-43) [inline README.md:237]: remove this if not needed

**Review by ishita-43 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (katarianikita2003) [reply README.md]: Only 237 Knife_hit?

File: assets/images/8_Puzzle.png
Comment (ishita-43) [inline assets/images/8_Puzzle.png:1]: remove this if not needed

File: assets/images/Screenshot 2024-06-05 220940.png
Comment (ishita-43) [inline assets/images/Screenshot 2024-06-05 220940.png:1]: delete if not needed

**Review by ishita-43 — CHANGES_REQUESTED** [review]

**Review by ishita-43 — CHANGES_REQUESTED** [review]
You need to add your game entry in main Readme.md file of the GameZone Repository.
Refer other entries for the format.

**Review by ishita-43 — APPROVED** [review]
Approved!!

**Pull request merged**